### PR TITLE
167 set admin role

### PR DESCRIPTION
**Pull request by FilipHasson**
Closes: #167 
Fixes:
Related: 

---

**Describe the pull request:**

The bot should now be able to change which role is treated as the botomir-admin role if server admins wish to change it
---

Checklist:

- [x] Run `npm run lint` and ensure linter passes
- [x] Run `npm run test` and ensure tests pass
- [x] Verify that the changes work as expected on Discord
- [x] Update documentation / not applicable
- [x] Update changelog / not applicable

### CODE REVIEW

File: test/bot/commands/admin/setBotomirAdminRole.tests.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const source = require('rfr');
+
+const { setBotomirAdminRole } = source('bot/commands/admin');
+
+describe('test setBotomirAdminRole', () => {
```
Comment (SoorajModi) [inline test/bot/commands/admin/setBotomirAdminRole.tests.js]: Bless ur soul for adding tests

**Review by SoorajModi — APPROVED** [review]

**Comment by SoorajModi** [comment]
Also check off the last two points in the PR template, even if its n/a

File: bot/commands/admin/setBotomirAdminRole.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+const source = require('rfr');
+
+const { sendMessage, lookupRoleName } = source('bot/utils/util');
+
+function setBotomirAdminRole(message, args, config) {
+    if (!lookupRoleName(message.guild, args.join())) {
```
Comment (MarshallAsch) [inline bot/commands/admin/setBotomirAdminRole.js:6]: add this line before this and use the variable instead

```
const roleName = args.join(' ');
```

**Review by MarshallAsch — CHANGES_REQUESTED** [review]

File: bot/commands/admin/setBotomirAdminRole.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        return sendMessage(message.channel, 'Error: that role does not exist!');
+    }
+    return config.setAdminRole(args.join())
+        .save()
+        .then(() => sendMessage(message.channel, `The admin role has been updated to ${args.join()}`))
```
Comment (MarshallAsch) [inline bot/commands/admin/setBotomirAdminRole.js:11]: same here

File: bot/commands/admin/setBotomirAdminRole.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+const source = require('rfr');
+
+const { sendMessage, lookupRoleName } = source('bot/utils/util');
+
+function setBotomirAdminRole(message, args, config) {
+    if (!lookupRoleName(message.guild, args.join())) {
+        return sendMessage(message.channel, 'Error: that role does not exist!');
+    }
+    return config.setAdminRole(args.join())
```
Comment (MarshallAsch) [inline bot/commands/admin/setBotomirAdminRole.js:9]: same here

File: bot/commands/admin/setBotomirAdminRole.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
+const source = require('rfr');
+
+const { sendMessage, lookupRoleName } = source('bot/utils/util');
+
+function setBotomirAdminRole(message, args, config) {
+    if (!lookupRoleName(message.guild, args.join(' '))) {
```
Comment (MarshallAsch) [inline bot/commands/admin/setBotomirAdminRole.js:6]: I maintain my previous comment about pulling this into a variable

**Review by MarshallAsch — CHANGES_REQUESTED** [review]

**Review by MarshallAsch — APPROVED** [review]

**Pull request merged**